lib/CryptolTC.z3

```diff
@@ -75,6 +75,24 @@
   (cryBool true)
 )
 
+
+(declare-fun cryPrimeUnknown (Int) Bool)
+
+(define-fun cryPrimeTable ((x Int)) Bool
+  (or (= x 2)
+  (or (= x 2)
+  (or (= x 3)
+  (or (= x 5)
+  (or (= x 7)
+  (cryPrimeUnknown x))))))
```

Comment:
I wonder if we should bother with this at all (i.e., maybe just leave it as uninterpreterd?)  I suspect in the cases where this is used, the primes are typically much larger...   Also we have 2 twice.

Reply:
> I wonder if we should bother with this at all (i.e., maybe just leave it as uninterpreterd?)

A good question. I only bothered with this because `cryExp` also goes through the effort of defining a table, so I followed suit here:

https://github.com/GaloisInc/cryptol/blob/4485d702e3c0b863afa22a6aafc696180c9bb877/lib/CryptolTC.z3#L276-L281

If we decide to remove the table for `cryPrime`, we probably should remove it for `cryExp` as well. (I'm not really sure how to test whether these are useful or not.)

> Also we have 2 twice.

Oops! I'll fix that.

Reply:
Fair enough, you've convinced me. I've removed `cryPrimeTable`.